### PR TITLE
Readme Updates

### PR DESCRIPTION
**Pull request by akshaaatt**
We have a lot of features which can take off the app to a whole new dimension!

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -54,6 +55,17 @@ MusicBrainz for Android is a mobile app for viewing MusicBrainz data on the Andr
...
 	  
 Got **something interesting** you'd like to **ask or share**? Start a discussion at `#metabrainz` IRC channel on libera.chat.
+
+## Roadmap
+
+Proposed future plans for the app are as follows:
+
+- Introduce Jetpack Compose to the app.
+- Support more entities for search.
+- Ability to add entities natively from the app.
```
Comment (amCap1712) [inline Readme.md:65]: MB API does not support it currently so we can't do this.

File: Readme.md [hunk 2]
```diff
@@ -54,6 +55,17 @@ MusicBrainz for Android is a mobile app for viewing MusicBrainz data on the Andr
...
+- Showcase ListenBrainz functionalities to discover a wide range of users based on musical similarities.
```
Comment (amCap1712) [inline Readme.md:66]: We should rephrase this to say, consider and add other *Brainz powered features.

Comment (akshaaatt) [reply Readme.md:66]: Sounds good!

File: Readme.md (hunk 1 above)
Comment (akshaaatt) [reply Readme.md:65]: Right. Now that I will be diving into the mb project soon, I think we could add this. But yes, for now let's get rid of this.
